src/language-css/parser-postcss.js

```diff
@@ -208,26 +214,78 @@ function parseNestedCSS(node) {
           node.source
         );
       }
+
+      return node;
     }
-    if (node.type === "css-atrule" && typeof node.params === "string") {
+
+    if (
+      node.type === "css-atrule" &&
+      typeof node.params === "string" &&
+      node.params.trim().length > 0
+    ) {
+      if (
+        node.name.toLowerCase() === "counter-style" ||
+        node.name.toLowerCase().endsWith("keyframes") ||
+        node.name.toLowerCase() === "page" ||
+        node.name.toLowerCase() === "font-feature-values"
+      ) {
+        return node;
+      }
+
       if (node.name === "warn" || node.name === "error") {
         node.params = {
           type: "media-unknown",
           value: node.params
         };
-      } else {
-        if (node.params.includes("#{")) {
-          // Workaround for media at rule with scss interpolation
-          return {
-            type: "media-unknown",
-            value: node.params
-          };
+
+        return node;
+      }
+
+      if (node.name === "extend") {
+        node.selector = parseSelector(node.params);
+        delete node.params;
+
+        return node;
+      }
+
+      if (node.name === "at-root") {
+        if (/^(\(\s*(without|with)\s*:\s*.+\s*\)$)/.test(node.params)) {
```

Comment:
`/^\(\s*(without|with)\s*:[\s\S]+\)$/`?

Can there be whitespace at the start and end? Or is that something the parser has taken care of?

Reply:
@lydell valid with any whitespace (`\t`, `\n`, `  ` and etc)

Reply:
I don't understand what you mean?

Reply:
@lydell valid:
```scss
@at-root 

(

without

: 

media

) {
    color: red;
}
```

Reply:
Let me rephrase:

1. What do you think of my Regex suggestion? `/^\(\s*(without|with)\s*:[\s\S]+\)$/`
2. Should there `\s*` at the start and end? `/^\s*\(\s*(without|with)\s*:[\s\S]+\)\s*$/`